rb/spec/integration/selenium/webdriver/bidi/browsing_context/browsing_context_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require_relative '../../spec_helper'
+
+module Selenium
+  module WebDriver
+    class BiDi
+      describe BrowsingContext, exclusive: {browser: %i[chrome firefox]} do
+        it 'can create a browsing context for given id' do
+          reset_driver!(web_socket_url: true) do |driver|
+            id = driver.window_handle
+            browsing_context = BrowsingContext.new(driver: driver, browsing_context_id: id)
+            expect(browsing_context.id).eql? id
+          end
+        end
+
+        it 'can create a window' do
+          reset_driver!(web_socket_url: true) do |driver|
+            browsing_context = BrowsingContext.new(driver: driver, type: :window)
+            expect(browsing_context.id).not_to be_nil
+          end
+        end
+
+        it 'can create a window with a reference context' do
+          reset_driver!(web_socket_url: true) do |driver|
+            browsing_context = BrowsingContext.new(driver: driver, type: :window,
+                                                   reference_context: driver.window_handle)
+            expect(browsing_context.id).not_to be_nil
+          end
+        end
+
+        it 'can create a tab' do
+          reset_driver!(web_socket_url: true) do |driver|
+            browsing_context = BrowsingContext.new(driver: driver, type: :tab)
+            expect(browsing_context.id).not_to be_nil
+          end
+        end
+
+        it 'can create a tab with a reference context' do
+          reset_driver!(web_socket_url: true) do |driver|
+            browsing_context = BrowsingContext.new(driver: driver, type: :tab, reference_context: driver.window_handle)
+            expect(browsing_context.id).not_to be_nil
+          end
+        end
+
+        it 'can navigate to a url' do
+          reset_driver!(web_socket_url: true) do |driver|
+            browsing_context = BrowsingContext.new(driver: driver, type: :tab)
+
+            info = browsing_context.navigate url: url_for('/bidi/logEntryAdded.html')
+
+            expect(browsing_context.id).not_to be_nil
+            expect(info.navigation_id).nil?
```

Comment:
It doesn't look like an RSpec expectation, does it actually fail the test?

Every expectation is written as 

```ruby
expect(...).to match(...)
```

So in this case it should be:


```suggestion
            expect(info.navigation_id).not_to be_nil
```